src/info/mod.rs

```diff
@@ -73,93 +65,15 @@ impl std::fmt::Display for Info {
-        }
-
-        if let Some(created_info_value) = self.created.get(&self.disabled_fields) {
-            self.write_styled_info_line(f, &self.created.title(), &created_info_value, true)?;
-        }
-
-        if let Some(languages_info_value) = self.languages.get(&self.disabled_fields) {
-            self.write_styled_info_line(f, &self.languages.title(), &languages_info_value, false)?;
-        }
-
-        if let Some(dependencies_info_value) = self.dependencies.get(&self.disabled_fields) {
-            self.write_styled_info_line(
-                f,
-                &self.dependencies.title(),
-                &dependencies_info_value,
-                true,
-            )?;
-        }
-
-        if let Some(authors_info_value) = self.authors.get(&self.disabled_fields) {
-            self.write_styled_info_line(f, &self.authors.title(), &authors_info_value, false)?;
-        }
-
-        if let Some(last_change_info_value) = self.last_change.get(&self.disabled_fields) {
-            self.write_styled_info_line(
-                f,
-                &self.last_change.title(),
-                &last_change_info_value,
-                true,
-            )?;
-        }
-
-        if let Some(contributors_info_value) = self.contributors.get(&self.disabled_fields) {
-            self.write_styled_info_line(
-                f,
-                &self.contributors.title(),
-                &contributors_info_value,
-                true,
-            )?;
-        }
-
-        if let Some(repo_info_value) = self.repo.get(&self.disabled_fields) {
-            self.write_styled_info_line(f, &self.repo.title(), &repo_info_value, true)?;
-        }
-
-        if let Some(commits_info_value) = self.commits.get(&self.disabled_fields) {
-            self.write_styled_info_line(f, &self.commits.title(), &commits_info_value, true)?;
-        }
-
-        if let Some(lines_of_code_info_value) = self.lines_of_code.get(&self.disabled_fields) {
-            self.write_styled_info_line(
-                f,
-                &self.lines_of_code.title(),
-                &lines_of_code_info_value,
-                true,
-            )?;
-        }
-
-        if let Some(size_info_value) = self.size.get(&self.disabled_fields) {
-            self.write_styled_info_line(f, &self.size.title(), &size_info_value, true)?;
-        }
-
-        if let Some(license_info_value) = self.license.get(&self.disabled_fields) {
-            self.write_styled_info_line(f, &self.license.title(), &license_info_value, true)?;
+        for info_field in self.info_fields.iter() {
+            if let Some(info_field_value) = info_field.get(&self.disabled_fields) {
+                self.write_styled_info_line(
+                    f,
+                    &info_field.title(),
+                    &info_field_value,
+                    info_field.should_color(),
+                )?;
+            }
```

Comment:
Just a nitpick to consider: `.filter_map` could probably be used to avoid an indentation level